lib/widgets/page_header.dart

```diff
@@ -23,12 +23,16 @@ class PageHeader extends StatelessWidget {
             crossAxisAlignment: CrossAxisAlignment.start,
             mainAxisSize: MainAxisSize.min,
             children: [
-              Row(
-                mainAxisAlignment: MainAxisAlignment.end,
-                children: [
-                  ...buttons,
-                  const SizedBox(width: 24),
-                ],
+              SingleChildScrollView(
+                scrollDirection: Axis.horizontal,
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: [
+                    const SizedBox(width: 50),
+                    ...buttons,
+                    const SizedBox(width: 24),
+                  ],
+                ),
```

Comment:
this is supposed to be in the separate branch too because this is not part of a fix for #61 (text scaling) but for #62 (top bar)